src/e2e/position.rs

```diff
@@ -286,8 +286,8 @@ fn test_remove_position() {
 
 #[test]
 fn test_position_within_current_tick() {
-    let alice = test_env::get_account(0);
-    test_env::set_caller(alice);
+    let deployer = test_env::get_account(0);
```

Comment:
In case you use only one account, it would be better to name it based on the action/role instead of `deployer`. Naming it as `deployer` in such a case makes sense only if you are testing admin entrypoints.